### PR TITLE
Add npstreams to conda-forge

### PR DESCRIPTION
**Pull request by LaurentRDC**

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
Please don't remove unconverted recipes

File: recipes/npstreams/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  requires:
+    - scipy >=1.0.0
+  imports:
+    - npstreams
+  commands:
+    - python -m unittest discover -b
+
+about:
+  home: https://github.com/LaurentRDC/npstreams
+  license: BSD
```
Comment (synapticarbors) [inline recipes/npstreams/meta.yaml]: I think this should be `license: BSD-3-Clause`

**Comment by LaurentRDC** [comment]
@synapticarbors I've also taken this opportunity to update the recipe to npstreams 1.5.2, which only supports python 3.6+.

**Comment by LaurentRDC** [comment]
Woops commented and closed at the same time

**Comment by synapticarbors** [comment]
looks like you need to update the hash for the newer version

**Comment by synapticarbors** [comment]
@LaurentRDC -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Comment by LaurentRDC** [comment]
@synapticarbors  Thanks a lot for your help!

**Pull request merged**